### ISSUE TITLE
Once authenticated, browser is not redirected to authGuardLoggedInURL path

### ISSUE THREAD

**Issue by dev-mansonthomas** [AnthonyNahas/ngx-auth-firebaseui#356]
### Bug Report or Feature Request (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?
MacOs Mojave 10.14.6 (18G95), Chrome  77.0.3865.90

### Versions
```
ng --version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 8.3.5
Node: 10.16.2
OS: darwin x64
Angular: 8.2.7
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.5
@angular-devkit/build-angular     0.803.5
@angular-devkit/build-optimizer   0.803.5
@angular-devkit/build-webpack     0.803.5
@angular-devkit/core              8.3.5
@angular-devkit/schematics        8.3.5
@angular/cdk                      8.2.0
@angular/cli                      8.3.5
@angular/fire                     5.2.1
@angular/flex-layout              8.0.0-beta.26
@angular/material                 8.2.0
@ngtools/webpack                  8.3.5
@schematics/angular               8.3.5
@schematics/update                0.803.5
rxjs                              6.5.3
typescript                        3.5.3
webpack                           4.39.2
```

### Repro steps

Once the login procedure is done successfully (either with google or basic login/password), the browser stays indefinitely on this screen.
The blue bar is animated, so the user expect something to happen.
The two horizontal grey bars seems to be links with no text (one to reach a page, but i don't know how the link is picked, the other seems a log out link)

![image](https://user-images.githubusercontent.com/5695269/65286955-aa6ae480-db41-11e9-9cc8-ac20181d9634.png)

### The log given by the failure
none

### Desired functionality
Get the browser redirected to the authGuardLoggedInURL path.
Or if it's not its use, give a way to redirect the browser to the welcome page of the app.

### Mention any other details that might be useful

https://github.com/dev-mansonthomas/RedCrossQuestFrontEnd/tree/master/src/

**Comment by dev-mansonthomas** [comment]
Also,

with toastMessageOnAuthSuccess: true,

the snackbar is not displayed.

**Comment by AnthonyNahas** [comment]
it's an issue related directly to firebase admin stuff!

Firebase does not enable duplicate accounts with the same email...

**Comment by dev-mansonthomas** [comment]
@AnthonyNahas 

> Firebase does not enable duplicate accounts with the same email...

Sorry I wasn't explicit enough : 

I set 
toastMessageOnAuthError: true
And I did have an error once (duplicate account with same email), but the toast was not displayed.

Neither the success toaster is with toastMessageOnAuthSuccess: true.


it was a side note which should have its own ticket.

The main point of this ticket is that when there's a success login, the browser is not redirected to "authGuardLoggedInURL", While I was expecting to do so. (for example, with a successful logon, redirect to the home page).

But maybe I didn't understand correctly the use of authGuardLoggedInURL
